native/src/modules/app/components/NavigationTypes.js

```diff
@@ -0,0 +1,109 @@
+// @flow
```

Comment:
should this file go into `constants` and have a lower case name as it exports no class?

Reply:
Hm not sure. Until now we have used uppercase naming all the time for flow type files. Moving to constants would make sense though. Will do in #105.

Reply:
I think in the long term it should be moved (or at least some parts of it) to the api-client to unify route configs.